index.html

```diff
@@ -1,155 +1,165 @@
-          </ul>
-          <button class="project-link" type="button">See Project</button>
+          <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
         </div>
+        <ul class="prog-lang">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javascript</li>
+        </ul>
+        <button class="project-link" type="button">See Project</button>
       </div>
-      <div class="card-works">
+    </div>
+    <div class="card-works card-reverse">
+      <img class="snapshot" src="work-snapshot-4.png" alt="project-snapshot" />
+      <div>
+        <h2 class="project-title">Multi-Post Stories</h2>
+        <div class="caption-container">
+          <div class="text">CANOPY</div>
+          <div class="dot">&middot</div>
+          <div class="text-light">Back End Dev</div>
+          <div class="dot">&middot;</div>
+          <div class="text-light">2015</div>
+        </div>
         <div>
-          <h2 class="project-title">Multi-Post Stories</h2>
-          <div class="caption-container">
-            <div class="text">CANOPY</div>
-            <div class="dot">&middot</div>
-            <div class="text-light">Back End Dev</div>
-            <div class="dot">&middot;</div>
-            <div class="text-light">2015</div>
-          </div>
-          <div>
-            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          </div>
-          <ul class="prog-lang">
-            <li class="lang">html</li>
-            <li class="lang">css</li>
-            <li class="lang">javascript</li>
-          </ul>
-          <button class="project-link" type="button">See Project</button>
+          <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
         </div>
-        <img class="snapshot" src="work-snapshot-4.png" alt="project-snapshot" />
+        <ul class="prog-lang">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javascript</li>
+        </ul>
+        <button class="project-link" type="button">See Project</button>
       </div>
-    </section>
-    <section class="about-me">
-      <div class="about-container">
+    </div>
+  </section>
+  <section class="about-me">
+    <div class="about-container">
+      <div>
         <h1 class="about-title">About Myself</h1>
         <div class="about-body">
           <p>
-            I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like
+            I'm a software developer! I can help you build a product, feature or website. Look through some of my work
+            and experience! If you like
             what you see and have a project you need coded, don't hesitate to contact me.
           </p>
```

Comment:
- [ ] If you notice, `LET'S CONNECT` mixes up with this `p`. This makes the website unfriendly to users 😞 . I, therefore, suggest that you consider fixing this bug to ensure that the design matches the Figma design 👌 . Below is how it appears.

- Your design 😎 

![overlapping](https://user-images.githubusercontent.com/99394374/213885842-5cd5b2fc-eea9-4483-b7d9-25bf6283bfec.PNG)